addons/account/tests/test_sequence_mixin.py

```diff
@@ -56,23 +56,23 @@ def test_sequence_change_date(self):
         """Change the sequence when we change the date iff it has never been posted."""
         # Check setup
         self.assertEqual(self.test_move.state, 'draft')
-        self.assertEqual(self.test_move.name, 'MISC/15-16/01/0001')
+        self.assertMoveName(self.test_move, 'MISC/15-16/01/0001')
```

Comment:
Note sure why you don't simply check the placeholder?

This test is not really relevant anymore; since we never set the sequence in draft anymore, we should never have to change it once it's posted. (from the docstring of this test)

Reply:
I interpreted the test more as a way to check that the sequence respects the dates rather than a way to check that the name is updated on a date change.

I think it's best not to erase this test entirely but i agree the placeholder can be tester instead of the name.